### PR TITLE
Dp/5 build UI scaffold for student card on dashboard

### PR DESCRIPTION
**Pull request by DanielPurdy509**
Student Card built it out with mock hard-coded data + some styling, added to dashboard
Blue_Ocean_Student_Card_Component

### CODE REVIEW

File: client/src/components/StudentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    <img className={StudentCardCSS.icon} src="https://img.icons8.com/?size=512&id=kDoeg22e5jUY&format=png"></img>
+                    <h3>{studentData.name}</h3>
+                </div>
+                <ul className={StudentCardCSS.studentInfoList}>
+                    <li className={StudentCardCSS.studentInfo}>Email: <b>{studentData.email}</b></li>
+                    <li className={StudentCardCSS.studentInfo}>Phone #: <b>{studentData.phone}</b></li>
+                    <li className={StudentCardCSS.studentInfo}>Cohort: <b>{studentData.cohort}</b></li>
+                    <li className={StudentCardCSS.studentInfo}>Status: <b className={statusStyle}>{studentData.status}</b></li>
+                    <li className={StudentCardCSS.studentInfo}>Created By: <b>{studentData.createdBy}</b></li>
+                    <li className={StudentCardCSS.studentInfo}>Technical Interview Data: <b>{studentData.interviewDate}</b></li>
```
Comment (ranegray) [inline client/src/components/StudentCard.jsx:52]: This should be technical interview date.

File: client/src/components/StudentCard.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import StudentCardCSS from "../css/StudentCard.module.css";
```
Comment (ranegray) [inline client/src/components/StudentCard.jsx:2]: Let's use object destructuring for cleaner JSX.

e.g. import { cardContainer, studentInfoCard, studentInfo, etc... } from <CSS FILE>

File: client/src/components/StudentCard.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    if(studentData.status == "Coding Challenge"){
+        statusStyle = StudentCardCSS.challenge;
+    }
+    if(studentData.status == "Prep Work"){
+        statusStyle = StudentCardCSS.prep;
+    }
+    if(studentData.status == "Done"){
+        statusStyle = StudentCardCSS.done;
+    }
```
Comment (ranegray) [inline client/src/components/StudentCard.jsx:37]: What if we made a status tag component that we can use on the card and in the table.

We can move this logic into that and all the CSS classes from the module to a new module for better reusability.

Let me know if you have any questions.

**Review by ranegray — CHANGES_REQUESTED** [review]
As a general note, lets colocate CSS with our components. 

Create a new folder for StudentCard which will have StudentCard.jsx and StudentCard.module.css

There will be no CSS folder going forward.

File: client/src/components/StudentCard.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        phone: "555-123-4567",
+        cohort: "MCSP-35",
+        status: "Technical Interview",
+        createdBy: "Rane Gray",
+        interviewDate: "June 6th, 2023 @ 10:00AM MST",
+        notes: {
+            attemptNum: 1,
+            date: "May 23rd, 2023",
+            narrative: "Jon performed decently on most topics. Struggled with accessing properties on an object"
+        }
```
[the remark this answers is not in the document]
Comment (jordan-carlson99) [reply client/src/components/StudentCard.jsx:21]: what would that look like for the database? csv? or should we consider making another table or column within users

**Comment by DanielPurdy509** [comment]
Summary of new changes:
- Reorganized components into folders with corresponding CSS files
- Created StatusTag component for use in the Student Card and elsewhere it is appropriate
- Refactored CSS stylings as requested 
- Changed the names of some of the student info properties as requested

**Review by ranegray — APPROVED** [review]

**Pull request merged**